onnxruntime/core/framework/session_options.cc

```diff
@@ -6,20 +6,19 @@
 
 namespace onnxruntime {
 
-bool HasSessionConfigEntry(const SessionOptions& options, const std::string& config_key) {
-  return options.session_configurations.find(config_key) != options.session_configurations.cend();
+bool SessionOptions::HasConfigEntry(const std::string& config_key) const noexcept {
+  return session_configurations.find(config_key) != session_configurations.cend();
 }
 
-const std::string GetSessionConfigOrDefault(const SessionOptions& options,
-                                            const std::string& config_key,
-                                            const std::string& default_value) {
-  if (!HasSessionConfigEntry(options, config_key))
+const std::string SessionOptions::GetConfigOrDefault(const std::string& config_key,
+                                                     const std::string& default_value) const noexcept {
+  if (!HasConfigEntry(config_key))
     return default_value;
 
-  return options.session_configurations.at(config_key);
+  return session_configurations.at(config_key);
```

Comment:
nit: could use [] operator since you're already checking for existence before or use the iterator returned by find.